Statistics.ipynb

```diff
@@ -227,136 +227,19 @@
-      "text/plain": [
-       "<Figure size 720x576 with 1 Axes>"
-      ]
-     },
-     "metadata": {
-      "needs_background": "light"
-     },
-     "output_type": "display_data"
     }
    ],
    "source": [
-    "from collections import Counter, defaultdict\n",
+    "import matplotlib.pyplot as plt     \n",
     "import pandas as pd\n",
-    "from PIL import Image\n",
```

Comment:
ma perche hai cancellato queste statistiche?

Reply:
Tutte le statistiche con riferimento alla sorgente dell'immagine sono state rimosse perché abbiamo rimosso l'informazione sulla sorgente dai json. Abbiamo ritenuto che potesse essere informazione sensibile.

Reply:
capisco, in ogni caso ti ricordo che quello e gia informazione pubblica, ha senso non metterlo per nuove immagini, ma non sono sicura ha senso toglierli per quelle che sono gia state pubblicate,